web/src/components/settingsmodal.jsx

```diff
@@ -76,25 +87,10 @@ const SettingButton = styled(Button).attrs({
   color: ${(props) => (props.active ? '#000' : '#999')};
 `;
 
-const LanguageSelect = ({ isOpen, onSelect }) => {
-  if (!isOpen) {
-    return null;
-  }
-
-  return (
-    <StyledLanguageSelectWrapper>
-      {Object.entries(LANGUAGE_NAMES).map(([key, language]) => (
-        <li key={key}>
-          <button onClick={() => onSelect(key)}>{language}</button>
-        </li>
-      ))}
-    </StyledLanguageSelectWrapper>
-  );
-};
-
 const SettingsView = () => {
   const { __, i18n } = useTranslation();
   const [languageSelectOpen, setLanguageSelectOpen] = useState(false);
+  const [selectedLanguage, setSelectedLanguage] = useState('English');
```

Comment:
We can't use english as the defualt here becuase then it will always be set to english before anyone changes the language (regardless of the language shown).

We need to get the language from i18n or local storage.